apps/web/pages/team/[slug]/[type].tsx

```diff
@@ -2,6 +2,7 @@ import type { GetServerSidePropsContext } from "next";
 
 import type { LocationObject } from "@calcom/core/location";
 import { privacyFilteredLocations } from "@calcom/core/location";
+import { getBookingFieldsWithSystemFields } from "@calcom/features/bookings/lib/getBookingFields";
```

Comment:
I had thought that Next.js should automatically remove this through treeshaking. Because it knows it's used in getServerSideProps only.

Found an article on it https://gedasfx.medium.com/how-does-next-js-10-handle-imports-c51b9e4451c6#:~:text=All%20imports%2C%20which%20are%20then%20used%20ONLY,will%20be%20sent%20over%20to%20the%20client.

Also, I spent some time verifying that it is being removed automatically, but because of `getBookingFieldsWithSystemFields` being imported elsewhere(some indirect dependency), it's still being imported.

I will do followup work on some approach to remove it. 
I also checked Next.js bundle analyzer stats and the payload isn't changing much from what we have already.

Reply:
Btw, while testing it out I found something else that we fix too, filed an issue for that [here](https://github.com/calcom/cal.com/issues/7276)

Reply:
Yes, I had already read that post, but in booking page it reduces -100kb when I tried async imports, maybe the reduction was for another reason.